trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_vortex_distribution.py

```diff
@@ -0,0 +1,1060 @@
+                za_te = np.concatenate([za_te , za_te ])            
+                xb_te = np.concatenate([xb_te , xb_te ])
+                yb_te = np.concatenate([yb_te ,-yb_te ])
+                zb_te = np.concatenate([zb_te , zb_te ]) 
+                
+                y_sw  = np.concatenate([y_sw,-y_sw ])
+                xc    = np.concatenate([xc ,xc ])
+                yc    = np.concatenate([yc ,-yc]) 
+                zc    = np.concatenate([zc ,zc ])
+                x     = np.concatenate([x , x ])
+                y     = np.concatenate([y ,-y])
+                z     = np.concatenate([z , z ])            
+
+        n_cp += len(xch)        
+
+        # ---------------------------------------------------------------------------------------
+        # STEP 7: Store wing in vehicle vector
+        # ---------------------------------------------------------------------------------------       
+        VD.XAH    = np.append(VD.XAH,xah)
+        VD.YAH    = np.append(VD.YAH,yah)
+        VD.ZAH    = np.append(VD.ZAH,zah)
+        VD.XBH    = np.append(VD.XBH,xbh)
+        VD.YBH    = np.append(VD.YBH,ybh)
+        VD.ZBH    = np.append(VD.ZBH,zbh)
+        VD.XCH    = np.append(VD.XCH,xch)
+        VD.YCH    = np.append(VD.YCH,ych)
+        VD.ZCH    = np.append(VD.ZCH,zch)            
+        VD.XA1    = np.append(VD.XA1,xa1)
+        VD.YA1    = np.append(VD.YA1,ya1)
+        VD.ZA1    = np.append(VD.ZA1,za1)
+        VD.XA2    = np.append(VD.XA2,xa2)
+        VD.YA2    = np.append(VD.YA2,ya2)
+        VD.ZA2    = np.append(VD.ZA2,za2)        
+        VD.XB1    = np.append(VD.XB1,xb1)
+        VD.YB1    = np.append(VD.YB1,yb1)
+        VD.ZB1    = np.append(VD.ZB1,zb1)
+        VD.XB2    = np.append(VD.XB2,xb2)                
+        VD.YB2    = np.append(VD.YB2,yb2)        
+        VD.ZB2    = np.append(VD.ZB2,zb2)    
+        VD.XA_TE  = np.append(VD.XA_TE,xa_te)
+        VD.YA_TE  = np.append(VD.YA_TE,ya_te) 
+        VD.ZA_TE  = np.append(VD.ZA_TE,za_te) 
+        VD.XB_TE  = np.append(VD.XB_TE,xb_te)
+        VD.YB_TE  = np.append(VD.YB_TE,yb_te) 
+        VD.ZB_TE  = np.append(VD.ZB_TE,zb_te)  
+        VD.XAC    = np.append(VD.XAC,xac)
+        VD.YAC    = np.append(VD.YAC,yac) 
+        VD.ZAC    = np.append(VD.ZAC,zac) 
+        VD.XBC    = np.append(VD.XBC,xbc)
+        VD.YBC    = np.append(VD.YBC,ybc) 
+        VD.ZBC    = np.append(VD.ZBC,zbc)  
+        VD.XC     = np.append(VD.XC ,xc)
+        VD.YC     = np.append(VD.YC ,yc)
+        VD.ZC     = np.append(VD.ZC ,zc)  
+        VD.X      = np.append(VD.X ,x)
+        VD.Y_SW   = np.append(VD.Y_SW ,y_sw)
+        VD.Y      = np.append(VD.Y ,y)
+        VD.Z      = np.append(VD.Z ,z)         
+        VD.CS     = np.append(VD.CS,cs_w)        
+
+    # ---------------------------------------------------------------------------------------
+    # STEP 8.1: Unpack aircraft fuselage geometry NOTE THAT FUSELAGE GOMETRY IS OMITTED FROM VLM
```

Comment:
is it still?

Reply:
Yes, I spoke to Kroo about it. I might revisit this in the future because I don't know all the tricks Drela has in his code